src/clj/rems/api/schema.clj

```diff
@@ -203,13 +203,25 @@
          :field/private s/Bool
          (s/optional-key :field/previous-value) schema-base/FieldValue))
 
+(s/defschema FormData
+  {:form/internal-name s/Str
+   :form/external-title LocalizedString})
+
 (s/defschema FormTemplate
   {:form/id s/Int
    :organization OrganizationOverview
-   :form/title s/Str
+   (s/optional-key :form/title) (rjs/field s/Str
+                                           {:deprecate true
+                                            :description "DEPRECATED, will disappear, use either internal name or external title as you need"})
+   :form/internal-name (rjs/field s/Str
+                                  {:description "The internal name of the form only visible to the administration."})
```

Comment:
Thanks for the docs! You can also use `(rjs/descibe s/Str "The internal name ...")` if you don't need to set any other metadata.

Reply:
Added a task to Ideas to refactor this. There seems to be a bunch.